org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -168,7 +168,7 @@ public Object get() {
 			return null;
 		}
 	}
-	
+
 
 	public Object get(boolean create) throws IllegalAccessException, IllegalArgumentException, InvocationTargetException, InstantiationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)